### ISSUE TITLE
Investigate adding checks for Mobile Accessibility Guidelines (MAG)

### ISSUE THREAD

**Issue by EmmaJP** [bbc/bbc-a11y#193]
The checks so far are HTML specific, covering responsive websites. Many of these map to the Mobile Accessibility Guidelines.

There are other Mobile Accessibility Guidelines not covered by HTML Guidelines, and some checks that might require using a mobile simulator or an actual mobile to run.

What MAG is already met for responsive HTML pages?
What MAG could be added to the automated checks?
What MAG could be added as semi-automated checks?
What MAG would have to be manual checks?
Is there a way to test on a simulated or actual mobile device?

**Comment by EmmaJP** [comment]
Revised MAG to be launched next week.

**Comment by EmmaJP** [comment]
bbc-a11y v2.0 aligns the tests to the updated MAG. The updated MAG applies to all digital content that can be viewed on mobile devices, including responsive web pages, and laptops. 

A cross-reference exercise was done to work out which HTML Guidelines mapped to which Mobile Accessibility Guidelines. And how they map to WCAG 2.0.

While many of the checks implemented are manual rather than automated, bbc-a11y does provide coverage and adding further automation should be possible and visible.
